bindings/pyroot/cppyy/CPyCppyy/src/ProxyWrappers.cxx

```diff
@@ -844,7 +844,6 @@ PyObject* CPyCppyy::BindCppObjectNoCast(Cppyy::TCppObject_t address,
     CPPInstance* pyobj =
         (CPPInstance*)((PyTypeObject*)pyclass)->tp_new((PyTypeObject*)pyclass, args, nullptr);
     Py_DECREF(args);
-    Py_DECREF(pyclass);
```

Comment:
For my understanding: The instance has a reference to the class object and therefore we can decref once here, right?

Reply:
We can decref it because `CreateScopeProxy` created a new reference to it, and we are done with it here.

Reply:
Well, actually we are done with it further down, that is why I moved the decref :)

Reply:
:+1: